src/buf/fixed/handle.rs

```diff
@@ -105,21 +92,24 @@ impl Deref for FixedBuf {
     type Target = [u8];
 
     fn deref(&self) -> &[u8] {
-        &self.buf
+        // Safety: The iovec points to a slice held  in self.buffers, to which no mutable reference exists.
```

Comment:
Thanks for the safety comments. I don't think I would have come up with these reasons as I'm not as focused on the self.buffers holding the ownership but I can't say the comments are incorrect.